components/LeagueCard/LeagueCard.test.tsx

```diff
@@ -85,4 +85,21 @@ describe('LeagueCard with custom league logo', () => {
     expect(leagueCardTitle).toBeInTheDocument();
     expect(leagueCardTitle).toHaveTextContent('League 1');
   });
+
+  it('should apply hover effect on mouse hover', () => {
+    render(
+      <LeagueCard
+        href="/leagues"
+        isEliminated={true}
+        survivors={11}
+        title="League 1"
+        totalPlayers={12}
+      />,
+    );
+    const leagueCard = screen.getByTestId('LeagueCard');
+    fireEvent.mouseOver(leagueCard);
+    expect(leagueCard).toHaveClass(
+      ' LeagueCard flex max-h-32 place-items-center gap-6 rounded-lg border bg-card p-4 text-card-foreground shadow-sm dark:border-zinc-800 hover:bg-zinc-800 transition',
```

Comment:
should this line just be the hover class, since your only looking for the component to have that.

Reply:
Initially, yes, but the `toHaveClass` wanted to see all of it for the `leagueCard`. Do you have any recommendations for Jest matchers that looks only for the hover?

Reply:
I don't currently. looks good